soccer/gameplay/root_play.py

```diff
@@ -192,7 +195,7 @@ def goalie_id(self):
 
     @goalie_id.setter
     def goalie_id(self, value):
-        self._goalie_id = None if value == -1 else value
+        self._goalie_id = value
```

Comment:
I think the C++ side still uses -1 to indicate that no goalie is selected ("None" from the dropdown in the gui).  See [MainWindow.cpp](https://github.com/robojackets/robocup-software/blob/3ed28930ceb81f271784b00eb55f36c87e28d7d9/soccer/MainWindow.cpp#L1167)  and [GameplayModule.cpp](https://github.com/robojackets/robocup-software/blob/master/soccer/gameplay/GameplayModule.cpp#L265).

Doing a search for `goalie_id` in the python code shows several places where `None` is handled separately from normal values.